.github/workflows/update-spincrystals.yml

```diff
@@ -0,0 +1,52 @@
+name: Spincrystals Update
+
+on:
+  schedule:
+    # Runs at 1:00 AM every Thursday every 2 weeks
+    - cron: '0 1 * * 4/2'
+  workflow_dispatch:  # Allows manual triggering
+
+jobs:
+  update-banners:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install missing packages
```

Comment:
Are these needed?

Reply:
I'm not really sure, borrowed this from another workflow as I'm not entirely sure how github workflows work. I guess the missing packages could be omitted though.

Reply:
You would need those packages if you would fetch via bash. Node/Bun already includes dependencies and git should already be installed on the runner